SeaPublicWebsite/Controllers/StaticPagesController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace SeaPublicWebsite.Controllers;
+
+public class StaticPagesController : Controller
+{
+    [HttpGet("/accessibility-statement")]
+    public IActionResult AccessibilityStatement()
+    {
+        return View("AccessibilityStatement");
+    }
+}
```

Comment:
We had no controller to handle this. This can be used for the future privacy policy page too.